content/reference/current/installation/docker/index.md

```diff
@@ -58,7 +58,7 @@ Make sure you are logged in using `docker login` with the username you provided
 
 ### Preparing the Cassandra database
 
-We will be using the official Cassandra image under the tag name `cassandra:3`.
+We will be using the official Cassandra image under the tag name `cassandra:4`.
```

Comment:
If we do that, we must also:
* update the installer to use Cassandra 4
* document the migration path